app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -23,46 +28,84 @@ const { NOT_CONNECTED, OPEN_APP, CONNECTED } = LEDGER_CONNECTION_STAGES
 
 type LedgerConnectionStage = $Values<typeof LEDGER_CONNECTION_STAGES>
 
+type LedgerPublicKey = { account: number, key: string }
+
 type Props = {
   progress: string,
+  publicKey: LedgerPublicKey,
   login: Function,
   connect: Function,
-  publicKey: ?string,
   error: ?string
 }
 
+type SelectOption = {
+  label: string,
+  value: string
+}
+
 type State = {
   ledgerStage: LedgerConnectionStage,
-  isLoading: boolean
+  isLoading: boolean,
```

Comment:
Btw, we can use trailing commas for cleaner diffs.
https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
Sure I am not opposed! In a separate PR I will update our prettier end eslint configurations.

Reply:
tracking here https://github.com/CityOfZion/neon-wallet/issues/1754